frontend/packages/client/src/networks.js

```diff
@@ -13,8 +13,8 @@ const networksConfig = {
     walletDiscovery: 'https://fcl-discovery.onflow.org/testnet/authn',
     walletDiscoveryApi: 'https://fcl-discovery.onflow.org/api/testnet/authn',
```

Comment:
will need client side for Dapper Wallet connection
`config({"discovery.wallet.method": "POP/RPC"})
`